src/utilities.jsx

```diff
@@ -0,0 +1,11 @@
+export const counter = {
+  count: 0,
+};
+
+export const IncrementCounter = () => ({
+  type: "counterIncrease",
```

Comment:
inside actions folder create a file constant.js and put all string and numeric value as a constant for action type and initial state